DNN Platform/Website/Install/App_LocalResources/UpgradeWizard.aspx.resx

```diff
@@ -226,35 +226,29 @@
     <value>Your installation is utilizing Telerik in the following assemblies:</value>
   </data>
   <data name="TelerikInstalledAndUsedWarning.Text" xml:space="preserve">
-    <value>It appears that you have extensions that depended on Telerik. We are unable to remove the dependencies we've detected and listed above for you. However, please ensure that you have in fact properly patched Telerik and removed all DNN Platform distributed Telerik components per the guidance found &lt;a href="https://docs.dnncommunity.org/content/getting-started/setup/telerik-removal/index.html" target="_blank"&gt;here&lt;/a&gt;. Note you will NOT be able to upgrade to DNN 10 with those dependencies unless they are distributed with their own copy of Telerik.</value>
+    <value>It appears that you have extensions that depended on Telerik. We are unable to remove the dependencies we've detected and listed above for you. However, this upgrade process &lt;em&gt;will&lt;/em&gt; remove the known-vulnerable Telerik components. Please ensure you have a backup of the site before continuing with the upgrade process.</value>
```

Comment:
Proposed updated verbiage
```suggestion
    <value>It appears that you have extensions that depended on Telerik. We are unable to remove the dependencies we've detected and listed above for you. However, this upgrade process &lt;em&gt;will&lt;/em&gt; remove the known-vulnerable Telerik components. Please ensure you have a backup of the site before continuing with the upgrade process.  After the upgrade completes, it is possible that your website will stop functioning, manually restoring the Telerik.dll may allow you to restore operation until you can uninstall the impacted extensions.</value>
```

I'm open to other ideas, but just want to give at least "some" guidance as to what to do next

Reply:
works for me